app/src/Book.php

```diff
@@ -0,0 +1,16 @@
+<?php
+
+namespace App;
+
+/**
+ * Class book - class with book attributes
```

Comment:
Could this be rewritten to 'Book Class' and maybe explain that it is used as a DB entity